config/sprayproxy.yaml

```diff
@@ -35,6 +35,25 @@ spec:
             limits:
               memory: "128Mi"
               cpu: "500m"
+        - name: kube-rbac-proxy
+          #TODO is there a preference between the kube-rbac-proxy image between kubebuilder and brancz?  Going with "org" repo vs. personal repo for now.
+          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
```

Comment:
We actually have this productized in OpenShift:

registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.12

Reply:
update pushed